packages/vite/src/node/plugins/importAnalysisBuild.ts

```diff
@@ -466,15 +476,7 @@ export function buildImportAnalysisPlugin(config: ResolvedConfig): Plugin {
                   deps.size > 1 ||
                     // main chunk is removed
                     (hasRemovedPureCssChunk && deps.size > 0)
-                    ? `[${[...deps]
-                        .map((d) =>
-                          JSON.stringify(
-                            relativePreloadUrls
-                              ? path.relative(path.dirname(file), d)
-                              : d
-                          )
-                        )
-                        .join(',')}]`
+                    ? `__viteFile([${[...deps].join(',')}])`
```

Comment:
I found the name `__viteFile` to be a bit confusing. Maybe `__viteDepMap`? Or at least we might want to have a comment in the code to indicate it's an optimization.

Reply:
Renamed to `__viteMapDep`.